package.json

```
@@ -263,7 +263,7 @@
     "regenerator-runtime": "0.13.7",
     "remark-parse": "9.0.0",
     "rosetta": "1.0.0",
-    "rxjs": "7.0.0",
+    "rxjs": "^7.8.1",
```

We want to pin our versions

```suggestion
    "rxjs": "7.8.1",
```